data/mships.txt

```diff
@@ -3417,7 +3417,7 @@ ship "EM-28F Wombat"
 	explode "medium explosion" 25
 	explode "large explosion" 35
 	explode "huge explosion" 20
-	description `Espion-exclusive mining carrier, or rather a mobile mining and refinery station to extract and process resources on the go as due to its large cargo holds it may spend much of its time in flight to collect a full cargo of resources. Due to its size and slowness the vessel is less than ideal for mining and collecting resources itself but instead relys and a sizeable fleet of fighters and drones to mine and collect resource for it. To protect their miners from harm as they operate far away from the mothership the Wombat also comes with space for 4 pocket light warships to guard the miners.`
+	description `The Espion-exclusive mining carrier, or rather a mobile mining and refinery station to extract and process resources on the go. Due to its large cargo holds it may spend much of its time in flight to collect a full cargo of resources. Because of its size and slowness, the vessel is less than ideal for mining and collecting resources itself, but instead relys on a sizeable fleet of fighters and drones to mine and collect resources for it. To protect their miners from harm as they operate far away from the mothership, the Wombat also comes with space for 4 pocket light warships to guard the miners.`
```

Comment:
```suggestion
	description `The Espion-exclusive mining carrier, or rather a mobile mining and refinery station to extract and process resources on the go as due to its large cargo holds it may spend much of its time in flight to collect a full cargo of resources. Because of its size and slowness, the vessel is less than ideal for mining and collecting resources itself, but instead relies on a sizable fleet of fighters and drones to mine and collect resources for it. To protect their miners from harm as they operate far away from the mothership, the Wombat also comes with space for 4 pocket light warships to guard the miners.`
```